Tests/Unit/Model/PaymentInAdvancePaymentInformationTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Model\PaymentInAdvancePaymentInformation;
+
+class PaymentInAdvancePaymentInformationTest extends OxidEsales\TestingLibrary\UnitTestCase
+{
+    private $_sAmount = "130";
```

Comment:
This doesn't need do be a property either. 😄 If you just use those values once, put them into the test directly.

Reply:
👍